stdlib/2.7/sqlite3/dbapi2.pyi

```diff
@@ -2,14 +2,19 @@
 #
 # NOTE: This dynamically typed stub was automatically generated by stubgen.
 
-from typing import Any
+from typing import Any, Union, List
+from numbers import Integral as _Integral
```

Comment:
The "as _Integral" is actually not needed. In stubs, imported modules and classes (etc.) are not exported -- unless you use the "as blah" syntax. So this does the opposite of what you want.